vision/keypoints/matching.go

```diff
@@ -31,7 +31,7 @@ func rangeInt(u, l, step int) []int {
 // MatchingConfig contains the parameters for matching descriptors.
 type MatchingConfig struct {
 	DoCrossCheck bool `json:"do_cross_check"`
-	MaxDist      int  `json:"max_dist"`
+	MaxDist      int  `json:"max_dist"` // TODO(RSDK-586): unit (px)?
```

Comment:
MaxDist is Max Hamming Distance, which represents the difference in bits between two binary strings. maybe just call it `max_dist_bits`